docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

```diff
@@ -383,6 +383,14 @@ Since :sinceVersion:`16.2.2: /etc/bareos-webui/configuration.ini` you are able t
    ; Default: 120000 milliseconds
    filetree_refresh_timeout=120000
 
+   ;------------------------------------------------------------------------------
+   ; THEME SETTINGS
+   ;------------------------------------------------------------------------------
+   [theme]
+   ; Possible values: bareos, ukraine
+   ; Default: bareos
```

Comment:
The default at the moment is actually "ukraine", but that will be changed back to "bareos" at some point in the future.

Reply:
> What is the slate theme about?

It's mentioned in the PR description.

> You want to mention it also?

No, it's experimental and working but not finished yet.

Reply:
> As a lot of files in the different theme directories are identical: how about linking the identical files to the default (bareos) theme?

No, better keep them separate. Something might change in future in one or the other. This is just the beginning of having themes.

> Probable also rename the bareos theme to `default`?

Yeah, we could do that.

Reply:
> The documentation should represent the current status. Changing the default will require changing the code. The documentation can than also be changed.

Will change that in https://github.com/bareos/bareos/pull/1281/commits/0151964fd574986a94de2525d1bb2aa623be2929 so we can simply git revert to the actual default at some point in time.

Reply:
> > The documentation should represent the current status. Changing the default will require changing the code. The documentation can than also be changed.
> 
> Will change that in [0151964](https://github.com/bareos/bareos/commit/0151964fd574986a94de2525d1bb2aa623be2929) so we can simply git revert to the actual default at some point in time.

Changed here: https://github.com/bareos/bareos/pull/1281/commits/c06180bcc0d1ad5fd80e84b8dee489679375d0cc

Reply:
Can we resolve this conversation?